example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>8.0</string>
+  <string>9.0</string>
```

Comment:
We may want to bump this to `10.0`, since that's [the minimum version that the native SDK supports](https://github.com/radarlabs/radar-sdk-ios/blob/c8e061084142fffaa4d66df76fe59737a0a63167/RadarSDK.xcodeproj/project.pbxproj#L813).